perun-core/src/main/java/cz/metacentrum/perun/core/api/AttributesManager.java

```diff
@@ -95,6 +95,9 @@
 
 	public static final String ATTRIBUTES_REGEXP = "^[-a-zA-Z0-9.]+([:][-a-zA-Z0-9.]+)?$";
 
+	public static final String[] ENTITY_TYPES = {"facility", "facility:cluster", "resource", "member_resource", "member_group",
```

Comment:
Please remove ``facility:cluster`` attribute type and all other references to it (NS_* constants). It's not used and never was. Thank you.